web/client/components/data/identify/viewers/HiddenViewer.jsx

```diff
@@ -0,0 +1,19 @@
+const React = require('react');
+const {Alert } = require('react-bootstrap');
+const HTML = require('../../../../components/I18N/HTML');
+
+class HiddenViewer extends React.Component {
+
+
+    render() {
+        return (
+            <div id="container">
+                <Alert bsStyle={"danger"}>
+                    <h4><HTML msgId="noFeatureInfo"/></h4>
+                </Alert>
+            </div>
+        );
+    }
+}
+
+module.exports = HiddenViewer;
```

Comment:
The hidden viewer and it's test is not needed anymore

Reply:
When looking at option 2) of the GitHub(#6101) issue. I use the Hidden viewer to render the error message Alert when calling `getViewers()` in MapInfoUtils. How would I go around this?

Reply:
When layers are filtered on the beginning (excluding not visible, not queriable), this happens automatically.
As you can see if I click and the layer is not visible, the error message appears.
![image](https://user-images.githubusercontent.com/1279510/98266235-59f35800-1f8a-11eb-8566-aaef078a56fc.png)

Reply:
probably this was the misundersanding :smile:

Reply:
<img width="1272" alt="Screenshot 2020-11-06 at 09 56 05" src="https://user-images.githubusercontent.com/30922279/98335582-4db4dc80-2016-11eb-8a06-6a492e5c5edb.png">
This is what it should be, correct?